src/main/java/com/company/ScrambleStrings.java

```
@@ -0,0 +1,48 @@
+package com.company;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScrambleStrings {
+    public static List<String> scrambleStrings(int left, int right, List<String> strings) {
+        List<String> result = new ArrayList<>();
```

You could throw an exception when strings is empty

---

Also when it is null - in this case you might get NullPointerException